docs/buildah-containers.md

```diff
@@ -36,12 +36,43 @@ Displays only the container IDs.
 ## EXAMPLE
 
 buildah containers
+```
+CONTAINER ID  BUILDER  IMAGE ID     IMAGE NAME                       CONTAINER NAME
+29bdb522fc62     *     3fd9065eaf02 docker.io/library/alpine:latest  alpine-working-container
+c6b04237ac8e     *     f9b6f7f7b9d3 docker.io/library/busybox:latest busybox-working-container
+```
 
 buildah containers --quiet
+```
+29bdb522fc62d43fca0c1a0f11cfc6dfcfed169cf6cf25f928ebca1a612ff5b0
```

Comment:
I thought `buildah containers --quiet` should truncate `IMAGE ID`. Am I wrong?

Reply:
'buildah containers --quiet" shows only the container id, I don't believe it was ever designed to show the short version of the id, but will let @nalind correct me on that.

Reply:
My bad. I meant `CONTAINER ID`.

From the below code it looks like even if `nontruncate` is false `CONTAINER ID` will never be truncated.

https://github.com/projectatomic/buildah/blob/fe2de4f491147e5063d12ff80431d80d42a37a0d/cmd/buildah/containers.go#L81-L82

Reply:
That's correct.  I expected the output of `--quiet` to get piped to 'xargs buildah rm/rmi' more often than not.  This was back when we weren't as good about handling truncated IDs for images, so it made a lot of sense to not require a `--notruncate=false` to get back values that we could use for that case.  Since we're already defaulting to `--notruncate` for `--quiet`, I'm not really sure of a good approach for adding the ability to have the output truncate IDs.

Reply:
Thanks @nalind, I just wanted to be sure.